Linux/Car_Client/chronoscomm.cpp

```diff
@@ -796,6 +796,28 @@ bool ChronosComm::decodeMsg(quint16 type, quint32 len, QByteArray payload, uint8
         emit monrRx(monr);
     } break;
 
+    case ISO_MSG_ACCM: {
+        qDebug() << "ACCM RECEIVED";
+        chronos_ACCM accm;
+        VByteArrayLe vb(payload);
+
+        accm.actionID = vb.vbPopFrontUint16();
+        accm.actionType = vb.vbPopFrontUint16();
+        accm.actionTypeParam1 = vb.vbPopFrontUint32();
+        accm.actionTypeParam2 = vb.vbPopFrontUint32();
+        accm.actionTypeParam3 = vb.vbPopFrontUint32();
```

Comment:
Could add a "// TODO " here to show that we still need to do the actual configuration based on the ACCM contents

Reply:
Which I think is a separate ticket

Reply:
Yes me too